README.md

```diff
@@ -68,6 +68,8 @@ Some useful command line parameters are:
 - `--server.port=9090` changes the http port (from default 8080) to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
+- `--bom-base.use-clearly-defined=false` disables the clearly-defined lookup,
```

Comment:
Wondering if `clearly-defined=false` would be more in line with the `scan-licenses=false` option.

(My suggestion is to drop the "use-" prefix.)

Reply:
use `harvest-clearly-defined` now.